presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,458 @@ Except for the limit on queued queries, when a resource group runs out of a reso
 
-* ``schedulingWeight`` (optional): weight of this sub-group. See above.
-  Defaults to ``1``.
+* ``schedulingWeight`` (optional): The weight for the resource group when the parent group uses the "weighted"
+  scheduling policy. A higher weight means that the group gets a larger share of the parent group's
+  resources.
 
-* ``jmxExport`` (optional): If true, group statistics are exported to JMX for monitoring.
+* ``jmxExport`` (optional):  If set to true, the statistics of the resource group will be exported via JMX.
   Defaults to ``false``.
 
-* ``perQueryLimits`` (optional): specifies max resources each query in a
-  resource group may consume before being killed. These limits are not inherited from parent groups.
-  May set three types of limits:
+* ``subGroups`` (optional): list of sub-groups.A list of sub-groups within the resource group.
+  Each sub-group can have its own set of properties.
 
-    * ``executionTimeLimit`` (optional): Specify an absolute value (i.e. ``1h``)
-       for the maximum time a query may take to execute.
+.. _scheduleweight-example:
 
-    * ``totalMemoryLimit`` (optional): Specify an absolute value (i.e. ``1GB``)
-       for the maximum distributed memory a query may consume.
+Scheduling Weight Example
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    * ``cpuTimeLimit`` (optional): Specify an absolute value (i.e. ``1h``)
-       for the maximum CPU time a query may use.
+Schedule weighting is a method of assigning a priority to a resource. Sub-groups
+with a higher scheduling weight are given higher priority. For example, to
+ensure timely execution of scheduled pipelines queries, weight them higher than
+adhoc queries.The scheduling weight in PrestoDB's Resource Groups is used when the parent group's
+scheduling policy is set to "weighted". It determines how resources are distributed
+among the subgroups. A higher weight means that the group gets a larger share of the parent group's resources.
 
-* ``workerPerQueryLimit`` (optional): specifies the minimum number of workers that have to
-  be available for each query. Intended to be used in elastic clusters where number of workers
-  varies over time.
+Here's an example:
 
-* ``subGroups`` (optional): list of sub-groups.
+Let's say we have a root resource group ``global`` with two subgroups: ``engineering`` and ``marketing``.
+The ``engineering`` subgroup has a scheduling weight of 3, and the ``marketing`` subgroup has a
+scheduling weight of 1. In this setup, the engineering subgroup will get 75% of the
+parent group's resources (because 3 is 75% of the total weight of 4), and the "marketing" subgroup will
+get 25% of the parent group's resources (because 1 is 25% of the total weight of 4).
+
+This allows you to prioritize certain subgroups over others in terms of resource allocation.
+In this example, queries from the "engineering" subgroup will be prioritized over queries
+from the "marketing" subgroup.
+
+... literalinclude:: schedule-weight-example.json
```

Comment:
Did you intend for this line and the next one to display as is in the documentation? 
<img width="799" alt="Screenshot 2023-09-05 at 3 21 31 PM" src="https://github.com/prestodb/presto/assets/7013443/ad08114a-844e-473f-8115-94d90302b741">